test-case-reporting/static/build-list.html

```diff
@@ -0,0 +1,17 @@
+<html ⚡>
+<head>
+  <title>Builds</title>
```

Comment:
```suggestion
  <title>Latest Builds</title>
```

Reply:
Build-list is for any build list (it's meant to be modular). I'll add a title field to the passed object.

Reply:
Done adding the title field, PTAL and let me know what you think.